distribution/lib/Standard/Base/0.0.0-dev/src/Any.enso

```diff
@@ -159,7 +156,7 @@ type Any
        Arguments:
        - that: The value to compare `self` against.
 
-       To have `>` properly defined, a type must have an associated ordered comparator.
+       To have `>` properly defined, a type must have an associated comparator.
```

Comment:
Now, isn't it that all types now have a comparator defined? So the `>` operator will work for all of them, it just may throw `Incomparable_Values`.

In that case I would rephrase this completely, maybe something along these lines:
```suggestion
       To be comparable, a custom object must have an associated comparator which will return `Ordering.Less/Greater` for unequal values. Otherwise, this will raise `Incomparable_Values` error.
```

Reply:
This is exactly how it works: `Default_Comparator` is for every object, by default. It just throws `Incomparable_Values` if you try to compare `x < y` that do not define ordering. Good idea.